src/org/gk/model/InstanceUtilities.java

```diff
@@ -1394,4 +1402,92 @@ else if (instance.getSchemClass().isa(ReactomeJavaConstants.Pathway))
 	//public static String encodeLineSeparators(String text) {
 	//    return text.replaceAll(FileAdaptor.LINE_END + "", "<br>");
 	//}
+
+     /**
+      * Determines if an instance is a drug or contains a drug (in the case of EntitySets and Complexes).
+      *
+      * @param instance
+      * @throws Exception
+      * @returns boolean
+      */
+     public static boolean isDrug(GKInstance instance) throws Exception {
+    	 if (instance == null)
+    		 return false;
+
+    	 SchemaClass schemaClass = instance.getSchemClass();
+    	 // Check if instance is a drug.
+    	 if (schemaClass.isa(ReactomeJavaConstants.Drug))
+    		 return true;
+
+    	 List<String> validClasses = Arrays.asList(ReactomeJavaConstants.EntitySet,
+												   ReactomeJavaConstants.Complex);
+    	 // Check id instance is an EntitySet or Complex.
+    	 if (validClasses.stream().noneMatch(schemaClass::isa))
+			 return false;
+    	 
+    	 // If EntitySet has a "hasMember" or "hasCandidate" attribute then recursively iterate over them.
+    	 // ifHasInstance()
+    	 Set<GKInstance> members = InstanceUtilities.getContainedInstances(instance,
```

Comment:
Please change members to another name, e.g. containedInstances

Reply:
Fixed in [4779567](https://github.com/reactome/CuratorTool/pull/23/commits/477956733a5f9958693bca874094cb1914bcd684).